definitions/contested/json/FixedLists/FixedLists.json

```diff
@@ -2278,40 +2278,47 @@
     "ListElement": "Expert Evidence",
     "DisplayOrder": 16
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "FR_s_caseDocumentType",
+    "ListElementCode": "Statement/Affidavit",
+    "ListElement": "Statement/Affidavit",
```

Comment:
my understanding is that nothing to do with the fixed list is changing?

Reply:
I noticed a discrepancy in the list provided. Chased it up and turns out they want an addition type.